server/routes/calendar.js

```diff
@@ -1,7 +1,9 @@
 const utilities = require('../helpers/utilities')
 const asyncMiddleware = require('../middleware/asyncMiddleware')
 
-module.exports = (logger, calendarService, calendarOvertimeService, notificationService, userAuthenticationService) => (router) => {
+module.exports = (logger, calendarService, calendarOvertimeService, notificationService, userAuthenticationService) => (
+  router,
+) => {
```

Comment:
Clearly this is a linting change, is the line too long to not remove the brackets around the returned functions params:
```
module.exports = (logger, calendarService, calendarOvertimeService, notificationService, userAuthenticationService) => router => {
```

Reply:
Done